### DISCUSSION TITLE
Better helm chart documentation for different releases

### DISCUSSION THREAD

**Discussion by shenrie** [apache/superset#18320] [Q&A / Help]
For whatever reason, I cannot install any version of superset except 1.0. I have cloned the superset repo and have tried various configuration to install using helm chart, but I always end up with version 1.0 and with this initialization error: 
ERROR [root] Error: Can't locate revision identified by '07071313dd52'

Because the associated migration file(s) is not present on the running kubernetes pod, because I end up with an earlier version. However, I do not know how the alembic_version table was updated with this version without the corresponding files. 

I have tried various versions of this command ( not in this order):
```

helm upgrade --install --values ../my-values.yaml superset ./helm/superset
helm upgrade --install --values ../my-values.yaml superset ./helm/superset --version 0.3.6
helm upgrade --install --values my-values.yaml superset ./superset-0.3.6.tgz 

```

To further add to my confusion, within the running superset UI it does not appear to display the right version.
<img width="385" alt="image" src="https://user-images.githubusercontent.com/768977/130887594-b2c3e123-eadf-4135-85a1-531441d557e3.png">

The only way I can see what versions are installed is by running helm ls

shenrie@Stephens-MBP superset % helm ls                                                                   
```
NAME           	NAMESPACE   	REVISION	UPDATED                             	STATUS  	CHART                 	APP VERSION
docker-registry	default 1       	2020-12-17 14:51:38.071211 -0700 MST	deployed	docker-registry-1.10.0	2.7.1      
jenkins        	default	1       	2021-07-06 11:43:37.322186 -0700 MST	deployed	jenkins-3.4.0         	2.289.1    
superset       	default	1       	2021-08-25 18:47:48.758689 -0700 MST	deployed	superset-0.3.6        	1.0   

```

What am I missing? And why does the superset documentation not have more clear guidance on how to update from version 1.0 to 1.x?

**Comment by nytai** [comment]
This is configurable https://github.com/apache/superset/blob/db11c3e6c829dbc4f103b58767cd2ca187a24b30/helm/superset/values.yaml#L143

As for the version displayed in the UI, only official release builds will have the UI showing the correct version

**Comment by shenrie** [comment]
> This is configurable
> 
> https://github.com/apache/superset/blob/db11c3e6c829dbc4f103b58767cd2ca187a24b30/helm/superset/values.yaml#L143
> 
> As for the version displayed in the UI, only official release builds will have the UI showing the correct version

Thanks for the reminder. I think the thing that adds to the confusion is that I would not have expected the "latest" tag to still be set to the 1.0 image, when there are at least two more recent official releases, 1.1 and 1.2, that I am aware of.

Changing the tag in my values file did the trick and 1.2 is showing in the UI, but that doesn't explain why the official 1.0 release did not display, unless you are also saying that 1.0 is not an official release. 

I do think that calling this out somewhere in the documentation might be helpful, since the doc seems to only assume a fresh install and nothing about how to do upgrades or maintenance. Thanks again!
